viewer/src/store.ts

```diff
@@ -49,7 +49,7 @@ const gaiaViewer = {
       accessibleSpaceMap: false,
       noFactionFill: false,
       flatBuildings: false,
-      highlightRecentActions: false,
+      highlightRecentActions: true,
```

Comment:
set to false

we could configure it so it reads from a .env file: https://cli.vuejs.org/guide/mode-and-env.html#example-staging-mode

eg: `highlightRecentActions: !!process.env.VUE_APP_highlightRecentActions`